mollie-payments-for-woocommerce/includes/mollie/wc/payment/RefundLineItemsBuilder.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
```

Comment:
It would be helpful if the classes had some sort of documentation, describing what they represent.

Reply:
Done